### PR TITLE
Release/3.1.1

### PR DESCRIPTION
**Pull request by and3r817**

### CODE REVIEW

File: Assets/BidMachine/Api/BannerRequest.cs [hunk 1]
```diff
@@ -16,7 +17,7 @@ public BannerSize GetSize()
             return client.GetSize();
         }
 
-        public string GetAuctionResult()
+        public AuctionResult GetAuctionResult()
```
Comment (staskochkin) [inline Assets/BidMachine/Api/BannerRequest.cs:20]: Don't just change the return type of the object in the public API. Let's deprecate 
```csharp
public string GetAuctionResult()
```
and add new method:
```csharp
public AuctionResult GetAuctionResultObject()
```

File: Assets/BidMachine/Api/InterstitialRequest.cs [hunk 2]
```diff
@@ -11,7 +11,7 @@ public InterstitialRequest(IAdRequest client)
             this.client = client;
         }
 
-        public string GetAuctionResult()
+        public AuctionResult GetAuctionResult()
```
Comment (staskochkin) [inline Assets/BidMachine/Api/InterstitialRequest.cs:14]: Same here

File: Assets/BidMachine/Api/RewardedRequest.cs [hunk 3]
```diff
@@ -11,7 +11,7 @@ public RewardedRequest(IAdRequest client)
             this.client = client;
         }
 
-        public string GetAuctionResult()
+        public AuctionResult GetAuctionResult()
```
Comment (staskochkin) [inline Assets/BidMachine/Api/RewardedRequest.cs:14]: Same here

File: Assets/BidMachine/Api/InterstitialRequest.cs (hunk 2 above)
Comment (and3r817) [reply Assets/BidMachine/Api/InterstitialRequest.cs]: ✅

File: Assets/BidMachine/Api/RewardedRequest.cs (hunk 3 above)
Comment (and3r817) [reply Assets/BidMachine/Api/RewardedRequest.cs]: ✅

**Review by staskochkin — APPROVED** [review]

**Review by dimasno1 — APPROVED** [review]

**Pull request merged**